samples/apps/copilot-chat-app/webapp/tests/chat.test.ts

```diff
@@ -1,51 +1,50 @@
-import { expect, test } from '@playwright/test';
+/* eslint-disable testing-library/prefer-screen-queries */
+import { test } from '@playwright/test';
+import * as util from './utils'
+import * as simpletests from './testsBasic'
+import * as plannertests from './testsPlanner'
+import { multiUserTest } from './testsMultiuser'
```

Comment:
nit: a little redundant
```suggestion
test suite for the functionality
```

Reply:
renamed the test to be a bit more specific